eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -68,6 +68,23 @@ jobs:
 
       - script: |
           node common/scripts/install-run-rush.js install
+          node common/scripts/install-run-rush.js lint $(rushFlags)
```

Comment:
We should run install and lint as two separate steps. On Windows I have noticed problems when you run multiple commands in the same script step.

Reply:
Ok thanks for the heads up

Reply:
done